### ISSUE TITLE
Porting backend to axum

### ISSUE THREAD

**Issue by pepoviola** [davidpdrsn/witter#22]
Hi @davidpdrsn,
                            First let me congrats for the announcement of `axum` 🎉🚀!! When I saw the announcement I remembered this repo and the streams and I think could be a nice exercise porting the backend to `axum`. 

Thanks and again awesome work!!

**Comment by davidpdrsn** [comment]
Yes totally! I have been thinking about doing a stream where we recreate the backend in axum 😊
